src/main/scala/it/unibo/authsim/library/dsl/otp/util/OTPHelpers.scala

```diff
@@ -69,4 +69,4 @@ object OTPHelpers:
   /**
    * A default implementation of an length generator (@see [[it.unibo.authsim.library.dsl.otp.builders.OTPBuilder.withPolicy]])
    */
-  implicit val generatorLength: OTPPolicy => Int = (policy: OTPPolicy) => Random.between(policy.minimumLength, policy.maximumLength + 1)
\ No newline at end of file
+  implicit val generatorLength: OTPPolicy => Int = (policy: OTPPolicy) => Random.between(policy.minimumLength, policy.maximumLength.getOrElse(policy.minimumLength + 10) + 1)
```

Comment:
10 is a magic number.

Reply:
In my opinion, it doesn't make sense to create a variable for 10 here.